### PR TITLE
Restore mention of popup-controlling features in window.open

### LINKED ISSUE #27357: window.open() has a partial list of features
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/Window/open

### What specific section or headline is this issue about?

Parameters -> windowFeatures

### What information was incorrect, unhelpful, or incomplete?

The list of features that can be passed to `window.open` is incomplete. It currently includes popup, width, innerWidth, height, innerHeight, left, screenX, top, screenY, noopener, noreferrer.

### What did you expect to see?

It omits the features mentioned at the linked spec: location, toolbar, menubar, resizable, scrollbars, status.

### Do you have any supporting links, references, or citations?

https://html.spec.whatwg.org/multipage/nav-history-apis.html

### Do you have anything more you want to share?

It's not clear to me which of these features are respected and which are legacy and should not be used. If these are mostly disregarded and *only* affect whether the browser treats a window as a popup, it should say so.

Also, the page does not communicate that innerwidth, innerheight, screenx, screeny aliases that exist only for legacy purposes.

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/api/window/open`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/Window/open
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/window/open/index.md
* Last commit: https://github.com/mdn/content/commit/39bb7174af0862fe3ef557562e720b5943f2fc06
* Document last modified: 2023-05-26T20:53:23.000Z

</details>

### PR DESCRIPTION
**Pull request by Josh-Cena**
These features got removed in https://github.com/mdn/content/pull/12425 but they are still part of the spec. Fix https://github.com/mdn/content/issues/27357.

### CODE REVIEW

File: files/en-us/web/api/window/open/index.md [hunk 1]
```diff
@@ -47,15 +47,9 @@ open(url, target, windowFeatures)
...
 
-        If `popup` is not enabled, and there are no window features declared, the new browsing context will be a tab.
-
-        > **Note:** Specifying any features in the `windowFeatures` parameter, other than `noopener` or `noreferrer`, also has the effect of requesting a popup.
-
-        To enable the feature, specify `popup` either with no value at all, or else set it to `yes`, `1`, or `true`.
-
-        Example: `popup=yes`, `popup=1`, `popup=true`, and `popup` all have identical results.
+        If `windowFeatures` is non-empty (except `noopener` and `noreferrer`), and `popup` is not set, the browser falls back to checking a few legacy features, which used to control UI features of the opened window. In modern browsers, they only have the effect of requesting a popup. They include: `location` and `toolbar` both being false or absent, or `menubar` being false or absent, or `resizable` being false, or `scrollbars` being false or absent, or `status` being false or absent. If _all_ conditions are not met, a new tab is opened. For example, `location=true,menubar=true,scrollbars=true,status=true,width=100` will open a new tab, effectively ignoring the `width` feature.
```
Comment (wbamberg) [inline files/en-us/web/api/window/open/index.md:52]: This is really hard to understand. I'm going to have a think about whether can can rephrase it in an easier way.

Comment (Josh-Cena) [reply files/en-us/web/api/window/open/index.md:52]: I agree it isn't easy.

- Empty `windowFeatures` (except `noopener` and `noreferrer`) never requests a popup. Otherwise
- `popup` being present always explicitly requests/forbids a popup. Otherwise
- `location` and `toolbar` both being false or absent always requests a popup. Otherwise
- `menubar` being false or absent always requests a popup. Otherwise
- `resizable` both being false always requests a popup. Otherwise
- `scrollbars` both being false or alsent always requests a popup. Otherwise
- `status` both being false or alsent always requests a popup.
- Failing all these, no popup is requested.

Comment (wbamberg) [reply files/en-us/web/api/window/open/index.md:52]: That is clearer (and much closer to the spec language.

Can we say this:

```
If `windowFeatures` contains any features other than `noopener`,`noreferrer`, or `attributionsrc`, and any of the following conditions apply, the window is opened as a popup:

- `popup` is true
- `location` and `toolbar` are both false or absent
- `menubar` is false or absent
- `resizable` is false
- `scrollbars` is false or absent
- `status` is false or absent

Otherwise the window is opened as a tab.
```

Am I missing cases here? One thing is that here:

> They include: `location` and `toolbar` both being false or absent, or `menubar` being false or absent, or `resizable` being false, or `scrollbars` being false or absent, or `status` being false or absent. If _all_ conditions are not met...

...it's hard to parse out the "They include" sentence into individual conditions, to meet up with "_all_ conditions".

Comment (Josh-Cena) [reply files/en-us/web/api/window/open/index.md:52]: That sounds great!

**Review by wbamberg — APPROVED** [review]
👍 thank you @Josh-Cena !
